applications/fishing-map/src/features/editor/editor.utils.ts

```diff
@@ -1,8 +1,23 @@
-import { Dataview } from '@globalfishingwatch/api-types/dist'
+import { kebabCase } from 'lodash'
+import { ColorCyclingType, Dataview, DataviewInstance } from '@globalfishingwatch/api-types/dist'
+import { Generators } from '@globalfishingwatch/layer-composer/dist'
 
 export const getDataviewInstanceFromDataview = (dataview: Dataview) => {
   return {
-    id: `${dataview.name}-${Date.now()}`,
+    id: `${kebabCase(dataview.name)}-${Date.now()}`,
     dataviewId: dataview.id,
   }
 }
+
+export const getActivityDataviewInstanceFromDataview = (
```

Comment:
The two functions in this file should probably move to `dataview.utils` since they are not editor-specific (anymore)